lib/operation.js

```diff
@@ -350,6 +350,40 @@ function normalize (normalize) {
   return this.normalise(normalize);
 }
 
+/**
+ * Perform contrast limiting adaptive histogram equalization
```

Comment:
Perhaps we could add a short description about using this operation to improve the clarity of an image, if only to help people find it (e.g. via keyword search)? Might be worth adding a link to https://en.wikipedia.org/wiki/Adaptive_histogram_equalization#Contrast_Limited_AHE too.